magnolia-voting-app/pom.xml

```diff
@@ -18,6 +18,11 @@
       <artifactId>magnolia-core</artifactId>
```

Comment:
Could you check this dependency? Do you really need this? If not, I think you don't need to import this.

Reply:
I have checked and realize it doesn't really need. I will delete it in this branch code